Presentation.Web/Controllers/API/V2/Internal/Organizations/OrganizationsInternalV2Controller.cs

```diff
@@ -6,25 +6,30 @@
 using Swashbuckle.Swagger.Annotations;
 using Presentation.Web.Controllers.API.V2.External.Generic;
 using Presentation.Web.Models.API.V2.Response.Shared;
+using Presentation.Web.Models.API.V2.Internal.Request.Organizations;
+using Presentation.Web.Models.API.V2.Response.Organization;
+using Presentation.Web.Models.API.V2.Internal.Response.Organizations;
 
 namespace Presentation.Web.Controllers.API.V2.Internal.Organizations
 {
     /// <summary>
     /// Internal API for the organizations in KITOS
     /// </summary>
-    [RoutePrefix("api/v2/internal/organizations/{organizationUuid}")]
```

Comment:
why remove the orgUuid from the Prefix? it's proceeds to be a part of every Route in the controller

Reply:
This is way back, but I think the patch endpoint didn't show up on the api until I switched to duplicating the uuid on each endpoint.

Reply:
It works now with a routeprefix